route.go

```diff
@@ -124,7 +124,7 @@ func cmdRouteRemove(args []string, home string) {
 	if len(args) == 0 {
 		log.Fatalf("must specify a <name> for the route to remove")
 	}
-	name := args[0]
+	names := args[0:]
```

Comment:
omit the first index?

```
names := args[:]
```

or, we can also use `args` directly instead of creating a new variable.